docs/en/operations/system_tables.md

```diff
@@ -89,9 +119,35 @@ Note that the amount of memory used by the dictionary is not proportional to the
 
 ## system.events {#system_tables-events}
 
-Contains information about the number of events that have occurred in the system. This is used for profiling and monitoring purposes.
-Example: The number of processed SELECT queries.
-Columns: 'event String' – the event name, and 'value UInt64' – the quantity.
+Contains information about the number of events that have occurred in the system. For example, in the table, you can find how many `SELECT` queries are processed from the moment of ClickHouse server start.
+
+Columns:
+
+- `event` ([String](../data_types/string.md)) — Event name.
```

Comment:
Name of event **kind**?

Reply:
Is there a difference?

Kind as a word can accumulate differently named events in a group. Here we have a textual identifier of event. We can call it name or id, but I think kind is not right word.